src/main/java/htsjdk/samtools/BAMFileReader.java

```diff
@@ -329,6 +329,29 @@ public ValidationStringency getValidationStringency() {
         return mCurrentIterator;
     }
 
+    /**
+     * Prepare to iterate through the SAMRecords in file order, starting at the given
+     * file pointer. The restrictions described for {@link #getIterator()} apply here.
```

Comment:
Yes - not needed in the latest version of the Hadoop-BAM client code. Well spotted!